### ISSUE TITLE
Plugin does not seem to recognize exclude-rules for gomnd

### ISSUE THREAD

**Issue by pauloavelar** [xxpxxxxp/intellij-plugin-golangci-lint#41]
I am not sure if this is related to how `golangci-lint` is called by the plugin or something specific to **gomnd**, but it is flagging magic numbers for files excluded in `issues.exclude-rules`.

When golangci-lint is called from the terminal, no issues are listed.

Exclusions:
```yaml
issues:
  exclude-use-default: true
  max-issues-per-linter: 0
  max-same-issues: 0
  new: false
  exclude-rules:
    - path: testutils
      linters:
        - gomnd
```

Screenshots:
![image](https://user-images.githubusercontent.com/1754076/98446655-c478d500-20fd-11eb-9740-c2c91f9a2902.png)

**Comment by xxpxxxxp** [comment]
Hi @pauloavelar , can I see your plugin config screeshot? It's in Settings - Tools - Go Linter

**Comment by pauloavelar** [comment]
Sure. There you go.
![image](https://user-images.githubusercontent.com/1754076/98468174-fea6ad00-21b7-11eb-845c-02fbaca62fc7.png)

This is the config file:
https://gist.github.com/pauloavelar/2586e46989b1e35d532decfab6990af8

**Comment by xxpxxxxp** [comment]
Got it, I can repro the issue with your config file, will fix soon.

**Comment by pauloavelar** [comment]
Can confirm this issue is fixed in the latest release (v1.4.7). Thanks for the quick fix.
And I realized you had to add another feature to avoid breaking some current expected behavior in #43, so kudos for that!

**Comment by xxpxxxxp** [comment]
Thanks for quick confirming!
